allennlp/training/trainer.py

```diff
@@ -1048,6 +1048,9 @@ def from_params(cls,  # type: ignore
         grad_clipping = params.pop_float("grad_clipping", None)
         lr_scheduler_params = params.pop("learning_rate_scheduler", None)
 
+        if isinstance(cuda_device, int) and cuda_device >= 0:
+            model = model.cuda(cuda_device)
```

Comment:
I'm not sure what the right thing to do is when you're using multiple GPUs - I'm not really familiar with how the multi-GPU code works.  Anyone have any idea?

Reply:
In the multiple GPU case, move the model to `cuda_device[0]` as is currently done in `Trainer.__init__` now (https://github.com/allenai/allennlp/blob/master/allennlp/training/trainer.py#L322)

Reply:
Thanks!